FirebaseAppCheck/Apps/FIRAppCheckTestApp/FIRAppCheckTestApp/AppDelegate.swift

```diff
@@ -75,6 +80,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
   }
 
+  func requestLimitedUseToken() {
+    AppCheck.appCheck().limitedUseToken { result, error in
+      if let result = result {
+        print("FAC limited-use token: \(result.token), expiration date: \(result.expirationDate)")
+      }
+
+      if let error = error {
+        print("Error: \(String(describing: error))")
+      }
+    }
+  }
```

Comment:
nit: Would you mind moving the new `requestLimitedUseToken` method above or below the provider tests? That way the provider tests will be grouped under `// MARK: App Check providers` and the new `requestLimitedUseToken` test could be separated under something like `// MARK: App Check API`. My argument for this is that the others are directly testing the providers whereas the new method is testing the top-level App Check API.

```suggestion

  // MARK: App Check API

  func requestLimitedUseToken() {
    AppCheck.appCheck().limitedUseToken { result, error in
      if let result = result {
        print("FAC limited-use token: \(result.token), expiration date: \(result.expirationDate)")
      }

      if let error = error {
        print("Error: \(String(describing: error))")
      }
    }
  }
```

Reply:
Sure, will moved